src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1594,7 +1594,6 @@ private void InitializeIncludedNodes()
             NodeFactory.AddTypeFactoryAndLoader(outputData.Type);
             NodeFactory.AddAlsoKnownAs(outputData.Type, outputData.AlsoKnownAs);
 
-            SearchModel?.Add(new CodeBlockNodeSearchElement(cbnData, LibraryServices));
```

Comment:
@Enzo707 after this change, can you test searching for code block node and see if it still works?

Reply:
@QilongTang if you look at the gif it still appearing in library search and seems to work fine. There's any other place to look at?

Reply:
yup, can you take a look at the in canvas search as well, just to make sure?

Reply:
@QilongTang it's failing within canvas search... I'm taking a look

Reply:
@Enzo707 Any update on this?

Reply:
@QilongTang sorry for the delay, please take a look.